src/main/java/com/redhat/darcy/salesforce/RelatedList.java

```diff
@@ -107,8 +115,22 @@ public boolean isEmpty() {
         return noRows.isDisplayed();
     }
 
+    public boolean canShowMore() {
+        return showMore.isDisplayed();
+    }
+
+    public void clickShowMoreLink() {
```

Comment:
What we might consider adding at some point is some interfaces in darcy-ui for tables that have this kind of "show more" functionality. Don't have to worry about this right away though.